extern/test_bmi_cpp/CMakeLists.txt

```diff
@@ -27,6 +27,7 @@ target_include_directories(testbmicppmodel PRIVATE ../bmi-cxx/ )
 set_target_properties(testbmicppmodel PROPERTIES VERSION ${PROJECT_VERSION})
 
 set_target_properties(testbmicppmodel PROPERTIES PUBLIC_HEADER include/test_bmi_cpp.hpp)
+set_target_properties(testbmicppmodel PROPERTIES CXX_VISIBILITY_PRESET default)
```

Comment:
From my perspective, this is a stop-gap toward actually export-marking the relevant exported symbols explicitly, to demonstrate the practice that we should be encouraging from real models that we integrate.

Reply:
Yes... to elaborate, when I added `add_subdirectory(test_bmi_cpp...)` to CMakeLists.txt, it was compiling with `-fvisibility=hidden` for reasons that weren't fully clear... it also wasn't fully clear why it _wasn't_ doing that when building test_bmi_cpp with cmake standalone. Arguably, including that flag and having the desired public symbols explicitly marked (as @PhilMiller suggests) are the best practice, though there's little or nothing in this test module that isn't intended to be visible.